app/jobs/delete_gyr_demo_info_job.rb

```diff
@@ -0,0 +1,21 @@
+class DeleteGyrDemoInfoJob < ApplicationJob
+  def perform(exempted_client_ids)
+    # Delete all non-exempted GYR clients
+    Client.includes(:intake).where(intake: { type: "Intake::GyrIntake" })
+          .where.not(id: exempted_client_ids).map(&:destroy)
+
+    # Delete all non-admin users
+    User.where.not(role_type: AdminRole::TYPE).map(&:destroy)
+
+    # Delete sites GYR sites with no clients
+    Site.where(national_overflow_location: false, processes_ctc: false)
+        .where.not(name: "GetCTC.org (Site)")
+        .where.missing(:clients).map(&:destroy)
```

Comment:
ohh ive never seen this .missing -- does that find things where there are 0 of that association? that's magical.

Reply:
right! its a new 6.1 thing I think ✨